MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -1489,16 +1489,16 @@ private string GetEncoderParam(EncoderPreset? preset, EncoderPreset defaultPrese
                      || string.Equals(videoEncoder, "hevc_qsv", StringComparison.OrdinalIgnoreCase) // hevc (hevc_qsv)
                      || string.Equals(videoEncoder, "av1_qsv", StringComparison.OrdinalIgnoreCase)) // av1 (av1_qsv)
             {
-                EncoderPreset[] valid_presets = [EncoderPreset.veryslow, EncoderPreset.slower, EncoderPreset.slow, EncoderPreset.medium, EncoderPreset.fast, EncoderPreset.faster, EncoderPreset.veryfast];
-
-                if (valid_presets.Contains(encoderPreset))
-                {
-                    param += " -preset " + encodingOptions.EncoderPreset;
```

Comment:
Just change this line to pass the encoderPreset as is should be fine if I understand it correctly

Reply:
This should likely fix it, yes.